### PR TITLE
Implement about page

### PR DESCRIPTION
**Pull request by acgrobman**
Also relocates the admin panel to a link on the about page.

### CODE REVIEW

File: public/locales/en/about.json [hunk 1]
```diff
@@ -1,3 +1,39 @@
 {
-    "heading": "About Us Page"
+    "pageHeading": "About Us",
+    "cehat": {
+        "heading": "South Gate CEHAT",
+        "part1": "The South Gate Community Environmental Health Action Team (CEHAT) is a group of local residents, business owners, community activists, and government employees whose mission is “to make a South Gate that is beautiful, safe, educated, represented, healthy, sustainable, prosperous, clean, and united.” The CEHAT was the first group in California to utilize the ",
```
Comment (CarsonHerness) [inline public/locales/en/about.json:5]: Was the South Gate CEHAT the first in California or CEHATs in general?

**Review by CarsonHerness — APPROVED** [review]

File: public/locales/en/about.json (hunk 1 above)
Comment (acgrobman) [reply public/locales/en/about.json:5]: [The CDC article](https://blogs.cdc.gov/yourhealthyourenvironment/2016/02/11/community-environmental-health-activism-in-south-gate-los-angeles-county-ca/) just says “This is the first time the PACE EH has been used in California to identify a city’s environmental issues and develop an action plan to tackle them.” Is the current phrasing indicating the wrong thing?

Comment (CarsonHerness) [reply public/locales/en/about.json:5]: I just wasn't sure if there were other "CEHATs", maybe say "The South Gate CEHAT was the first group ..."

But I think what you have is also fine

Comment (CarsonHerness) [reply public/locales/en/about.json:5]: I just wasn't sure if there were other "CEHATs", maybe say "The South Gate CEHAT was the first group ..."

But I think what you have is also fine

**Review by akampen — APPROVED** [review]

File: public/locales/en/about.json (hunk 1 above)
Comment (acgrobman) [reply public/locales/en/about.json:5]: I think I'm gonna leave it just so there's not two sentences in a row that both start the same way.

**Pull request merged**